spec/features/merchant_items/show_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe 'Merchant Items Show' do
+  before :each do
+    @merchant = Merchant.create!(name: "Angela's Shop")
+    @item_1 = @merchant.items.create!(name: "Jade Rabbit", description: "25mmx25mm hand carved jade rabbit", unit_price: 2500)
+  end
+
+  it 'displays its attributes' do
+
+    visit "/merchants/#{@merchant.id}/items/#{@item_1.id}"
+
+    expect(page).to have_content(@item_1.name)
+    expect(page).to have_content(@item_1.description)
+    expect(page).to have_content("$25.00")
+  end
+
+  it 'can edit the item' do
+
+    visit "/merchants/#{@merchant.id}/items/#{@item_1.id}"
+
+    within('#update_link') do
+      click_link
+    end
+
+    fill_in 'Name', with: "Moazinite Rabbit"
+    fill_in 'Description', with: "25mmx25mm Moazinite Rabbit"
+    fill_in 'Unit price', with: "4500"
```

Comment:
Add sad path test where certain fields are *not* filled in and the form can still successfully submit.

Reply:
Also add sad path where non numeric input is entered

Reply:
sad path test added for not filled in fields, non-numeric input is disallowed by html and validations which is tested